util/test/demos/vk/vk_test.cpp

```diff
@@ -1503,11 +1488,19 @@ VulkanWindow::~VulkanWindow()
   DestroySwapchain();
 
   {
-    for(size_t i = 0; i < ARRAY_COUNT(renderStartSemaphore); i++)
+    for(const auto &semaphore : renderStartSemaphore)
```

Comment:
The demos project is a little looser on coding style but I'd still prefer not to use `auto` here especially when the type is very simple `VkSemaphore`.